Tests/Unit/LoRaTools/LoRaCloudToDeviceMessageTest.cs

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace LoRaWan.Tests.Unit.LoRaWanTests
+namespace LoRaWan.Tests.LoRaTools.CommonAPI
```

Comment:
The namespace here doesn't match the convention to always start with `LoRaWan.Tests.Unit`.

Reply:
Fixed in 8fd7eb42de7b8974bc25bc5426b455fa768baf28